src/domain/landingPage/LandingPage.tsx

```diff
@@ -1,12 +1,77 @@
+import { IconPhoto } from 'hds-react';
 import React from 'react';
+import { useTranslation } from 'react-i18next';
 
+import myHelsinkiImage from '../../assets/images/png/myhelsinki-card.png';
+import tapahtumatImage from '../../assets/images/png/tapahtumat-hel-card.png';
+import Highlight from '../../common/components/highlight/Highlight';
+import useLocale from '../../hooks/useLocale';
+import Container from '../app/layout/Container';
+import FormContainer from '../app/layout/FormContainer';
 import PageWrapper from '../app/layout/PageWrapper';
 import Hero from './hero/Hero';
+import styles from './landingPage.module.scss';
+import ServiceCard from './serviceCard/ServiceCard';
 
 const LandingPage: React.FC = () => {
+  const locale = useLocale();
+  const { t } = useTranslation();
+
+  const myHelsinkiRoute = `https://www.myhelsinki.fi/${locale}`;
+  const tapahtumatHelRoute = `https://tapahtumat.hel.fi/${locale}/home`;
+
   return (
     <PageWrapper>
       <Hero />
+      <Container>
+        <FormContainer>
```

Comment:
FormContainer is little bit confusing here :)

Reply:
Its horrible name for it :D I implemented the component when it was only used in event form and didn't change the name. Maybe now is good time to rename it

Reply:
Removed FormContainer component and added same functionalities to Container component